core/trino-main/src/test/java/io/trino/sql/query/TestFilterHideInacessibleColumnsSession.java

```diff
@@ -51,44 +45,36 @@ public void testEnableWhenAlreadyEnabledByDefault()
     {
         FeaturesConfig featuresConfig = new FeaturesConfig();
         featuresConfig.setHideInaccessibleColumns(true);
-        SessionPropertyManager sessionPropertyManager = new SessionPropertyManager(new SystemSessionProperties(
-                new QueryManagerConfig(),
-                new TaskManagerConfig(),
-                new MemoryManagerConfig(),
-                featuresConfig,
-                new NodeMemoryConfig(),
-                new DynamicFilterConfig(),
-                new NodeSchedulerConfig()));
+        SessionPropertyManager sessionPropertyManager = createSessionPropertyManager(featuresConfig);
         assertThatNoException().isThrownBy(() -> sessionPropertyManager.validateSystemSessionProperty(SystemSessionProperties.HIDE_INACCESSIBLE_COLUMNS, "true"));
     }
 
     @Test
     public void testDisableWhenAlreadyDisabledByDefault()
     {
         FeaturesConfig featuresConfig = new FeaturesConfig();
-        SessionPropertyManager sessionPropertyManager = new SessionPropertyManager(new SystemSessionProperties(
-                new QueryManagerConfig(),
-                new TaskManagerConfig(),
-                new MemoryManagerConfig(),
-                featuresConfig,
-                new NodeMemoryConfig(),
-                new DynamicFilterConfig(),
-                new NodeSchedulerConfig()));
+        SessionPropertyManager sessionPropertyManager = createSessionPropertyManager(featuresConfig);
         assertThatNoException().isThrownBy(() -> sessionPropertyManager.validateSystemSessionProperty(SystemSessionProperties.HIDE_INACCESSIBLE_COLUMNS, "false"));
     }
 
     @Test
     public void testEnableWhenDisabledByDefault()
     {
         FeaturesConfig featuresConfig = new FeaturesConfig();
-        SessionPropertyManager sessionPropertyManager = new SessionPropertyManager(new SystemSessionProperties(
+        SessionPropertyManager sessionPropertyManager = createSessionPropertyManager(featuresConfig);
+        assertThatNoException().isThrownBy(() -> sessionPropertyManager.validateSystemSessionProperty(SystemSessionProperties.HIDE_INACCESSIBLE_COLUMNS, "true"));
+    }
+
+    private SessionPropertyManager createSessionPropertyManager(FeaturesConfig featuresConfig)
```

Comment:
Put that method extraction in a separate commit

Reply:
I knew you would say that...
done